topydo/lib/Config.py

```diff
@@ -311,6 +343,32 @@ def list_format(self):
         """ Returns the list format used by `ls` """
         return self.cp.get('ls', 'list_format')
 
+    def column_keymap(self):
+        """ Returns keymap and keystates used in column mode """
+        keystates = []
+
+        shortcuts = self.cp.items('column_keymap')
+        keymap_dict = dict(shortcuts)
+        for shortcut in shortcuts:
+            combo, action = shortcut
+            combo_list = re.split('(<[A-Z].+?>|.)', combo)[1::2]
+
+            combo_len = len(combo_list)
+            if combo_len > 1:
+                n = 1
+                while n < combo_len:
+                    keystates.append(''.join(combo_list[:n]))
+                    n += 1
```

Comment:
Implemented in 2ece281203a1386cd019a1bfa4bc19549a239da3 and f3b18f24337adfaff2482e2ed12c80db9b75753f.